src/redmodding.metadata.ts

```diff
@@ -9,6 +9,7 @@ export const REDMODDING_REQUIRED_DIR_FOR_GENERATED_FILES = path.join(`r6\\cache\
 export const REDMODDING_RTTI_METADATA_FILE_PATH = path.join(`tools\\redmod\\metadata.json`);
 
 export const V2077_LOAD_ORDER_DIR = path.join(`${V2077_DIR}\\Load Order`);
+export const V2077_MODLIST_PATH = path.join(`${V2077_DIR}\\modlist.txt`);
```

Comment:
Maybe throw this inside the LO dir?

Reply:
Either works. I figured since we are backing up load orders in the folder, it'd be good to keep the modlist.txt outside of it to keep it safe or something?

Reply:
Hmm, good point about duplicating the data. Although maybe we specifically _don't_ need to back this one up.. in which case it could be placed in `r6/cache/modded/`, so that there's a better chance it's in sync with the user maybe deleting the cache?

I don't think it's critical either way

Reply:
That... is a very good point. If it shouldn't be in the V2077 folder, then that would be the ideal place. I wasn't sure before hand. Maybe I will make that change later.